tests/test_bucketlist.py

```diff
@@ -0,0 +1,100 @@
+"""test_bucketlist.py."""
+import json
+from tests.test_setup import BaseTestCase
+
+
+class BucketListTestCase(BaseTestCase):
+    """This class contains tests for the bucketlist."""
+
+    URL = "/api/v1/bucketlists/"
+
+    def test_createBucketList(self):
+        """Test for successful creation of a bucketlist."""
+        payload = {'bucketlist_title': 'Visit Kenya'}
+        response = self.client.post(self.URL, data=json.dumps(payload),
+                                    headers=self.auth_header,
+                                    content_type="application/json")
+        self.assertEqual(response.status_code, 201)
+        self.assertEqual("Bucketlist created successfully", str(response.data))
+
+    def test_createBucketList_that_exists(self):
+        """Test for creation of a bucketlist that already exists."""
+        payload = {'bucketlist_title': 'Visit Paris'}
+        response = self.client.post(self.URL, data=json.dumps(payload),
+                                    headers=self.auth_header,
+                                    content_type="application/json")
+        self.assertEqual(response.status_code, 409)
+        self.assertIn("This bucketlist already exists", str(response.data))
+
+    def test_getBucketLists(self):
+        """Test for retrieval of all bucketlists."""
+        response = self.client.get(self.URL, headers=self.auth_header)
+        self.assertEqual(response.status_code, 200)
```

Comment:
Maybe check the response here contains a bucketlist eg. asserting the length of the object you're getting back